src/main/java/com/dungeoncrawler/Javiarenka/dungeonMapGenerator/MapController.java

```diff
@@ -0,0 +1,32 @@
+package com.dungeoncrawler.Javiarenka.dungeonMapGenerator;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+
+@Controller
+//@RequestMapping("generateMap") //wtedy wszystkie metody poniżej będą zaczynać się od /generateMap/...
+public class MapController
+{
+    MapGeneratorService service = new MapGeneratorService();
+
+    @RequestMapping(method = {RequestMethod.GET, RequestMethod.POST}, value = "/dungeonMap")
+    public String generateMap(@RequestParam int width, @RequestParam int height, Model model) throws IOException
```

Comment:
Zastanawiam się, czy nie można zrobić w całości tego generatora bez rzucania IOException - czyli bez zapisywania całego HTMLa do pliku, tylko z generowaniem jego thymeleafem na podstawie tablicy w pamięci.

Reply:
Jeśli generujemy mapę Springiem poprzez forma /createMap, to dzieje sie właśnie tak jak piszesz, tzn html generowany jest Thymeleafem "w locie". CSS natomiast jest updatowany JS-em.

Natomiast na potrzeby szybkiego debugowania, ciągle generuję sobie html metodą wymagającą IOException.